### PR TITLE
0.14.0

### PR DESCRIPTION
**Pull request by hjheath**
PR to trigger Travis build. Not planning to merge until the full release is approved (i.e. not a release candidate)

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://142-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://142-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://142-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://142-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,16 +6,20 @@ Breaking changes result in a different major. UI changes that might break custom
 
 ## [next-version]
 
+## [0.14.0-rc.2]
+
 ### Changed
 
 - Public: Document and face captures will be returned by callbacks as File or Blob instead of base64.
+- Internal: Callbacks are now linked to the flow rather than the Redux store.
```
Comment (rfreitas) [inline CHANGELOG.md]: this is helpful for us, but there might be something helpful for the use. We have now changed the behaviour of the `onComplete` callback, right?

Comment (hjheath) [inline CHANGELOG.md]: Fixed a bug where it would not fire if you did not capture both docs and live photos

**Comment by rfreitas** [comment]
Travis automatic deployment: https://142-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://142-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
@dylowicz how are we on this release?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://142-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**